### PR TITLE
Bookstore: Styling Project - Added styles to all components

### PR DESCRIPTION
**Pull request by kit0-0**
**Description:**

- Styled the book list to improve readability and create an appealing layout for displaying book information.
- Designed the container to provide a cohesive and visually pleasing structure for the entire bookstore page.
- Enhanced the appearance of the add book forms to make them more user-friendly.
- [🔗 Live Demo Link](https://kit0-0.github.io/bookstore-react-redux/)

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -3,42 +3,44 @@ import PropTypes from 'prop-types';
...
+                  <span className="fw-bold title">{title}</span>
                   <span className="text-primary">{author}</span>
                 </div>
                 <div className="book-buttons-container mt-3">
                   <Button variant="light  text-primary">Comments</Button>
                   <span className="mx-1">|</span>
-                  <Button variant="light text-primary">Remove</Button>
+                  <Button variant="light text-primary" onClick={() => onRemove(item_id)}>
+                    Remove
+                  </Button>
```
Comment (V-Blaze) [inline src/components/Book.js]: > Styling is **EXACTLY** the same as the one provided on [Zeplin](https://zeplin.io/)

> Your final design should match Zeplin's design (this includes all elements + the same styling)

- You have done great so far, :+1: :100: But there are some slight changes you need to make before moving to the next milestone. :+1: :+1: 
- [x] Looking at the project requirement as I pointed out above, it says your design need to be **Exactly** like the given design, so looking at the screenshot below, you can see that your texts are not properly aligned like the one on the provided design. I'll suggest you update his styling. :+1: :+1: 

- - Your design:
Selection_092

- - Provided design:
Selection_090

File: src/components/AddBook.js [hunk 2]
```diff
@@ -1,23 +1,72 @@
...
+            onChange={(e) => setAuthor(e.target.value)}
+          />
+        </Col>
+        <Col md={4}>
+          <Form.Control
+            type="text"
+            placeholder="Book Category"
+            value={category}
+            onChange={(e) => setCategory(e.target.value)}
+          />
```
Comment (V-Blaze) [inline src/components/AddBook.js]: - [ ] As I already pointed out above about replicating the exact design, your form layout needs a bit of adjustment, the form input and the button are all in the same line from the given design, I'll suggest you update your design to look like this too. :small_airplane: :smile: :wave: 

- - Your design:
Selection_091

- - Provided design:
Screenshot from 2023-07-24 15-54-27

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @kit0-0 , 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

giphy


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- Clean and organized code✔
-  Meaningful commit messages👋 
-  Descriptive pull request title✔

 ## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Gambit142 — APPROVED** [review]
Hi @kit0-0 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- All linter checks passed ✔️ 
- Good use of styled components ✔️ 
- Application look identical to that of the Zeplin template ✔️ 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**